gulpfile.js

```diff
@@ -61,6 +62,13 @@ function minifyJS() {
 	];
 }
 
+function lint(cb) {
+	return pump([
+		src(['**/*.js', '!node_modules/**'], { base: './' }),
+		eslint(),
+		eslint.format(),
+	], cb);
```

Comment:
Do you really want to not [fail on lint errors](https://github.com/PrismJS/prism/blob/662bbfe5828a0160d2035940469f8a9339fc3f22/gulpfile.js#L53)?

Reply:
I&nbsp;accidentally forgot to&nbsp;add&nbsp;it.